app/controllers/api/account.php

```diff
@@ -232,7 +232,6 @@
         $user->setAttributes($profile->getArrayCopy());
 
         $duration = $project->getAttribute('auths', [])['duration'] ?? Auth::TOKEN_EXPIRATION_LOGIN_LONG;
-
         $detector = new Detector($request->getUserAgent('UNKNOWN'));
         $record = $geodb->get($request->getIP());
         $expire = DateTime::formatTz(DateTime::addSeconds(new \DateTime(), $duration));
```

Comment:
is this variable being used? Else we can remove it

Reply:
You can either delete the variable or remove `DateTime::formatTz()` from it

Reply:
For all the expire variable comments, I checked once, its always used when we set cookie.

Reply:
Ill move them all near the cookies so its clear.